server/testdriver/core/src/main/java/org/infinispan/server/test/core/rollingupgrade/RollingUpgradeHandler.java

```diff
@@ -27,6 +32,8 @@ public class RollingUpgradeHandler {
    private ContainerInfinispanServerDriver toDriver;
 
    private RemoteCacheManager remoteCacheManager;
+   private RestClient[] fromRestClients;
```

Comment:
I think it might be better to move the node offset value into the handler. This way it knows what position it is in and which nodes should be OLD and which should be NEW. This way we only need a single collection of clients and the user doesn't have to know which is to or from.

Reply:
Updated now. I still have the client to specify the server offset, I think that might be useful for tests later, but it now automatically identifies if this is using the old or new version.

Reply:
Nice, yeah, this is more along with what I was thinking. Looks nice.